src/renderer/dx/DxRenderer.cpp

```diff
@@ -1125,6 +1136,13 @@ enum class CursorPaintType
     }
     case CursorPaintType::Outline:
     {
+        // DrawRectangle in antialiasing mode will straddle the pixels; fix that

```

Comment:
If we get around to doing the issue that lets people choose how much antialiasing they want (cleartype, grayscale, none), will this become a problem that we should watch out for?

Reply:
Fortunately, no! This actually forces the line to come out like it would if AA were off.

Reply:
Good.